src/main/java/de/samply/common/ldmclient/centraxx/LdmClientCentraxx.java

```diff
@@ -168,11 +169,18 @@ public QueryResult getResult(String location) throws LdmClientException {
     QueryResultStatistic queryResultStatistic = getQueryResultStatistic(location);
 
     if (queryResultStatistic != null && queryResultStatistic.getTotalSize() > 0) {
-      for (int i = 0; i < queryResultStatistic.getNumberOfPages(); ++i) {
+
+      int numberOfPages = queryResultStatistic.getNumberOfPages();
+      PercentageLogger percentageLogger = new PercentageLogger(logger, numberOfPages,
+          "getting results from centraxx...");
+
+      for (int i = 0; i < numberOfPages; ++i) {
         QueryResult queryResultPage = getResultPage(location, i);
         queryResult.getPatient().addAll(queryResultPage.getPatient());
       }
       queryResult.setId(queryResultStatistic.getRequestId());
+      percentageLogger.incrementCounter();
```

Comment:
should be inside the for loop

Reply:
Thank you! We copied it manually from Bitbucket and made a mistake (in the Bitbucket version, it was inside the loop)